### PR TITLE
Setup RAFI test repo structure and automate one sample test scenario

### PR DESCRIPTION
**Pull request by aaddepalli**
W-12007577 : Setup RAFI test repo structure and automate one sample test scenario

This is the first commit INS-Q1Q2 automation.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,293 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>RAFI-Core-SampleTestProject</artifactId>
```
Comment (lavanya948) [inline pom.xml:8]: Is it intentional to keep SampleTestProject for group & artifactId?

**Review by lavanya948 — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (aaddepalli) [reply pom.xml]: No.. I changed it

**Review by lavanya948 — APPROVED** [review]

**Review by lavanya948 — APPROVED** [review]

File: src/test/resources/setup/users-setup_default.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (lavanya948) [inline src/test/resources/setup/users-setup_default.json:1]: This file is not required for insurance as the credentials are not related to insurance org

File: src/test/resources/setup/users-setup_GS0.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "users" :[
+    {
```
Comment (lavanya948) [inline src/test/resources/setup/users-setup_GS0.json:3]: Please update user credentials with Q2 org user details

File: src/test/resources/setup/configuration.properties [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#
```
Comment (lavanya948) [inline src/test/resources/setup/configuration.properties:1]: Not required for insurance tests

**Review by lavanya948 — CHANGES_REQUESTED** [review]

File: src/test/resources/setup/configuration.properties (hunk 4 above)
Comment (aaddepalli) [reply src/test/resources/setup/configuration.properties:1]: Removed.

File: src/test/resources/setup/users-setup_GS0.json (hunk 3 above)
Comment (aaddepalli) [reply src/test/resources/setup/users-setup_GS0.json:3]: Fixed.

File: src/test/resources/setup/users-setup_default.json (hunk 2 above)
Comment (aaddepalli) [reply src/test/resources/setup/users-setup_default.json:1]: Updated.

**Review by lavanya948 — APPROVED** [review]

**Review by ankit-sharma-3 — APPROVED** [review]

**Pull request merged**